src/args.rs

```diff
@@ -10,42 +10,55 @@ use apca::api::v2::order;
 use chrono::NaiveDate;
 use chrono::NaiveDateTime;
 
-use num_decimal::Num;
+use clap::Args as ClapArgs;
+use clap::ArgEnum;
+use clap::ArgGroup;
+use clap::Parser;
+use clap::Subcommand;
 
-use structopt::clap::ArgGroup;
-use structopt::StructOpt;
+use num_decimal::Num;
 
 use uuid::Error as UuidError;
 use uuid::Uuid;
 
 
 /// A command line client for automated trading with Alpaca.
-#[derive(Debug, StructOpt)]
-#[structopt(about, version = env!("VERSION"))]
+#[derive(Debug, Parser)]
+#[clap(name = "apcacli")]
+#[clap(author = "Daniel Mueller <deso@posteo.net>")]
+#[clap(version = env!("VERSION"))]
+#[clap(about = "A command line tool for trading stocks on Alpaca (alpaca.markets).", long_about = None)]
```

Comment:
Shall we just use this string in the doc comment of the type and then remove the explicit setting here?

Reply:
Done.